src/Router.sol

```diff
@@ -151,10 +151,10 @@ contract Router is IRouter, EIP712, FeeGenerator {
         emit Paused();
     }
 
-    /// @notice Resume `execute` and `executeWithSignature` by pauser
-    function resume() external onlyPauser {
+    /// @notice Unpause `execute` and `executeWithSignature` by pauser
+    function unpause() external onlyPauser {
         paused = false;
```

Comment:
Wondering if `delete paused` is better?

Reply:
I think most cases to use delete variables are struct variables. For me, using assign false here is more suitable.

Reply:
hmm, I'm not sure if it's still the case that we can get a gas refund from `delete`. If it's not the case anymore, either way is fine.

Reply:
I like to maintain the original approach; it's uncommon to come across other protocols using the "delete" method to reset a uint256 variable. Even OpenZeppelin assigns "false" to the "pause" variable in their `Pausable.sol` implementation.

Reply:
There is no difference between `delete` and false assignment on gas refund.